docs/changelog.rst

```diff
@@ -6,6 +6,8 @@ Changelog
 
 develop
 -----------------
+* DataContext.get_docs_sites_urls now raises error if non-existent site_name is specified
+* Bugfix for the CLI command `docs build` ignoring the --site_name argument (#1378)
```

Comment:
Please note updated PR title and use for the changelog -- this issue was not contained to the CLI, but affected all build docs commands (including the UpdateDataDocs action)